ci_tools/process_results.py

```diff
@@ -0,0 +1,48 @@
+""" Script to process the output of numpydoc validator"
+"""
+import argparse
+import sys
+
+parser = argparse.ArgumentParser(description='Process the output of numpydoc validator')
+parser.add_argument('report', metavar='report', type=str,
+                        help='Report generated by numpydoc validator')
+parser.add_argument('summary', metavar='summary', type=str,
+                        help='Github step summary')
+args = parser.parse_args()
+
+warning_codes = ['EX01', 'SA01']
+
+errors = {}
+warnings = {}
+with open(args.report, 'r', encoding='utf-8') as f:
+    for line in f:
+        file_name, code, msg = line.split(':')
```

Comment:
@jalalium This command does not work in my branch. Any idea why? Does it fail if I add a function?